api/docs/reference/search_algorithm.md

```diff
@@ -0,0 +1,226 @@
+these _exactly_ which will bypass stemming and match exact word order. Title
+weighting is still applied in this case. This means that if the exact query text
+is found in the title of one document and the description of another, the
+title-matched document will appear ahead of the description-matched document.
+
+### Individual field querying
+
+This type of search is not commonly used. The Openverse frontend only surfaces
+one of the subset of fields that are queryable in this approach. It is only
+available if the `q` query parameter is excluded. In other words, even if the
+request includes these query parameters, _the API ignores these options if the
+`q` query parameter is present_. In that case, it will execute the
+[general "query" search](#general-query-searching) described above. Future work
+may surface the possibility for combining general querying and individual field
+querying.
+
+These are the fields currently supported for individual field querying:
+
+- Creator
+- Title
+- Tags
+
+> **Note**: The "tags" filter applies to the `name` field of the tags. No other
+> aspect of the tags is currently searchable.
+
+Each of these can be stacked. You can make a request that queries for a specific
+title by a specific creator. The following parameters would search only for
+works by "Claude Monet" where the word "madame" (or it's stemmed versions)
+appear in the title: `?creator=Claude Monet&title=madame`.
+
+As you can see, this is unique from the general query searching in that it
+allows you to apply separate queries for individual fields (hence the name
+"individual field querying"). The other notable difference from general querying
+is that the "description" field of the document is not available for individual
+field querying.
+
+## Document scoring
+
+Aside from the aforementioned weighting of document "title" matches, Openverse
+also includes one other attempt at scoring documents to improve search relevancy
+and quality. Future improvements to Openverse's search relevancy will most
+likely involve changes to how we score documents. For example, we may score
+documents higher if they are determined to be popular results in Openverse
+itself.
+
+### Provider supplied popularity
+
+Some providers supply a "popularity" rating for individual works. We ingest this
+data and calculate a normalised "popularity" score named `rank_feature`. Flickr
+is one of the providers that supplies this information. This data is used so
+that works that are popular on the provider side, are ranked higher in Openverse
+as well. The assumption here is that works that are popular on the provider's
+own website are likely higher quality and therefore more desirable results.
+Whether this has a significant impact on result relevancy or quality has not
+been measured, in part due to loose definitions of "relevancy" and "quality" and
+in part because we do not currently have tools for measuring user perception of
+a results relevancy or quality.
```

Comment:
I was going to say that we should add a note on the ["authority_boost"](https://github.com/WordPress/openverse-api/blob/2fb82a725b37f2c43c8ad2f2e8939047bb82ee2b/ingestion_server/ingestion_server/elasticsearch_models.py#L167), but I guess we only calculate it during the ingestion, and never use for search ranking? It seems that only the popularity boosting is used, which makes the images from providers who have popularity (Wikimedia) rank much higher than GLAM images 🤔 

https://github.com/WordPress/openverse-api/blob/2fb82a725b37f2c43c8ad2f2e8939047bb82ee2b/api/catalog/api/controllers/search_controller.py#L377

Reply:
Authority boosting is not currently enabled, that is correct.